### PR TITLE
Use iTunes API instead of Spotify

### PR DESCRIPTION
**Pull request by filmaj**
Since spotify is like, not public anymore.

FYI @audreyso

### CODE REVIEW

File: www/js/my-app.js [hunk 1]
```diff
@@ -114,32 +114,34 @@ function init() {
...
-  formData.type = 'track';
+  formData.media = 'music';
   $$('input').blur();
   myApp.showPreloader('Searching');
   $$.ajax({
     dataType: 'json',
     data: formData,
     processData: true,
-    url: 'https://api.spotify.com/v1/search',
+    url: 'https://itunes.apple.com/search',
```
Comment (devgeeks) [inline www/js/my-app.js]: You'll probably want to update the CSP in `index.html` as well for when not running in the developer app, etc.

File: www/index.html [hunk 2]
```diff
@@ -330,15 +330,15 @@
                   data-template="details"
                 >
                   <div class="item-media">
-                    <img width="80" src="{{this.album.images[0].url}}">
+                    <img width="80" src="{{this.artworkUrl100}}">
```
Comment (devgeeks) [inline www/index.html]: Shame iTunes doesn't have a higher res image than this :/

**Review by devgeeks — CHANGES_REQUESTED** [review]
Just update the CSP and this is good to go! Thanks so much for doing this!!!

File: www/index.html (hunk 2 above)
Comment (filmaj) [reply www/index.html]: Yeah, the options were 30, 60 and 100 :(

**Comment by filmaj** [comment]
Done. Also updated the README. This PR likely addresses #26?

**Review by surajpindoria — COMMENTED** [review]
Just played around with this and I think it looks good. The favorites list is a little messed up, but I don't think this is a blocker.

**Comment by macdonst** [comment]
Working good for me. Unfortunate about the album art size but I think we should merge this so folks can test their PG Day workshops properly.

**Pull request merged**